src/kaltura-player.js

```diff
@@ -326,6 +326,7 @@ class KalturaPlayer extends FakeEventTarget {
     this._localPlayer.destroy();
     this._eventManager.destroy();
     this._viewabilityManager.destroy();
+    RemotePlayerManager.destroy();
```

Comment:
it won't destroy the remote player without type, so I guess it should remove them all on destroy as I wrote in the previous comment, am I right?

Reply:
Good point. FIXED